src/components/Courses/components/CourseCard/CourseCard.module.css

```diff
@@ -0,0 +1,47 @@
+.courseCard {
+  display: flex;
+  margin-top: 2rem;
```

Comment:
Not a big deal, just an advice for the future: I recommend to use `%` or `px` for setting the width or height of elements since other items are not too effective or not supported by browsers